lib/install/config/webpacker.yml

```diff
@@ -34,6 +34,7 @@ development:
     host: localhost
     port: 8080
     https: false
+    hot_reloading: false
```

Comment:
Doesn't that point even further to the fact that the dev server should just be relegated to the live-reloading/hmr option? If you have team members who don't want/need that, then they can just rely on on-demand compilation, which is the default now anyway?